modules/claims_api/app/swagger/claims_api/v2/production/swagger.json

```diff
@@ -1912,6 +1912,20 @@
                                     "default": false,
                                     "nullable": true
                                   },
+                                  "specialIssues": {
+                                    "type": "array",
+                                    "nullable": true,
+                                    "uniqueItems": true,
+                                    "items": {
+                                      "maxItems": 2,
+                                      "additionalProperties": false,
+                                      "type": "string",
+                                      "enum": [
+                                        "POW",
+                                        "EMP"
+                                      ]
+                                    }
+                                  },
```

Comment:
@stiehlrod These are not going out to Production yet, correct?

Reply:
@rockwellwindsor-va There is a corresponding release note in review as well, so no, waiting until we are ready

Reply:
@rockwellwindsor-va good catch, thank you!